CHANGELOG.md

```diff
@@ -2,9 +2,32 @@
 
 ## unreleased
 
+## 3.0.0-beta.1 / 2024-10-09
+
+* [CHANGE] regexp `.` now matches all characters (performance improvement). #14505
 * [CHANGE] `holt_winters` is now called `double_exponential_smoothing` and moves behind the [experimental-promql-functions feature flag](https://prometheus.io/docs/prometheus/latest/feature_flags/#experimental-promql-functions). #14930
 * [CHANGE] API: The OTLP receiver endpoint can now be enabled using `--web.enable-otlp-receiver` instead of `--enable-feature=otlp-write-receiver`. #14894
+* [CHANGE] Prometheus will not add or remove port numbers from the target address. `no-default-scrape-port` feature flag removed. #14160
+* [CHANGE] Logging: the format of log lines has changed a little, along with the adoption of Go's Structured Logging package. #14906
+* [CHANGE]: Don't create extra `_created` timeseries if feature-flag `created-timestamp-zero-ingestion' is enabled. #14738
```

Comment:
I guess `* [FEATURE] SD: Support K8s sidecar containers in endpoint discovery. #14929` is still missing here?

Reply:
The commits from that PR do not mention "#14929" so it was missed.
I have asked about this on the issue.

Interested to know how you picked it up?

Reply:
I just saw the commit itself (and it was one of those where the commit description was the unedited result of the squashing :cry: )